docs/api/users.md

```diff
@@ -24,9 +24,8 @@ POST /users HTTP/1.1
 Name         | Type       | Description
 -------------|------------|------------
 id           | `string`   | **Optional** A unique identifier. If none is given, one is generated.
-firstName    | `string`   | **Required** The first name of the user.
-infix        | `string`   | **Required** The infix of the user.
-lastName     | `string`   | **Required** The last name of the user.
+profile      | `object`   | **Optional** The profile containing information about the user.
+groups       | `string[]` | **Optional** The groups this user is part of.
```

Comment:
Groups is not part of the profile change, I'd rather see this in a separate commit.